src/types/modules/react.d.ts

```diff
@@ -0,0 +1,6 @@
+import React from 'react';
+
+declare module 'react' {
```

Comment:
can you explain why we need this module argumentation?

Reply:
ah didn't know forwardRef cannot preserve the type info on a generic component. nice to know. thank you for the link!